ios/RNFirebase/config/RNFirebaseRemoteConfig.m

```diff
@@ -63,7 +73,7 @@ @implementation RNFirebaseRemoteConfig
         rejecter:(RCTPromiseRejectBlock)reject) {
     [[FIRRemoteConfig remoteConfig] fetchWithExpirationDuration:expirationDuration.doubleValue completionHandler:^(FIRRemoteConfigFetchStatus status, NSError *__nullable error) {
         if (error) {
-            reject(convertFIRRemoteConfigFetchStatusToNSString(status), error.localizedDescription, error);
+            reject(convertFIRRemoteConfigFetchStatusToNSString(status), convertFIRRemoteConfigFetchStatusToNSStringDescription(status), error);
         } else {
             resolve(convertFIRRemoteConfigFetchStatusToNSString(status));
```

Comment:
Could change to nil - doesn't need to resolve any sort of meaningful value really.